src/frontend/web_application/src/components/form/CheckboxFieldGroup/style.scss

```diff
@@ -1,10 +1,12 @@
 @import '../../../styles/common';
-@import '../../../styles/vendor/bootstrap_foundation-sites';
-
-$switch-height: 2rem !default;
 
 .m-switch-field-group {
+
   &__label {
     padding-left: map-get($co-form__spacing, 'small');
+
+    span {
```

Comment:
I suppose it's forcing TextBlock in displaySwitch to be inline (not easy to find).
It should not be like this. 
Do not use TextBlock maybe.